src/utils/constants/User.ts

```diff
@@ -0,0 +1,30 @@
+import { IUser } from 'types';
+import { Language, Themes } from 'utils/enums';
+
+export const User = {
```

Comment:
The same const User: IUser = {}

Reply:
constant User have string - firebaseKey and obj - initInfo
initInfo - using IUser, not User